views/admin/versions/versions.ejs

```diff
@@ -31,9 +31,10 @@ function deleteVersion(id) {
     $.ajax({
       url: "/version/"+id,
       type: "DELETE",
-      dataType: "text",
+      dataType: "json",
     })
         .done(function(response) {
+            console.log("Version deleted successfully!");
```

Comment:
If you have it, it would be nice to include ID and name (you may just have the ID) of what was deleted. When writing to the log file, it's important to take yourself out of the context of the code and, instead, change your perspective to someone trying to sort out a problem and looking through thousands of lines in a log file.